src/scripts/h5p-crossword-content.js

```diff
@@ -280,10 +280,15 @@ export default class CrosswordContent {
       return;
     }
 
+    const cells = this.table.getAnswers();
+    const focus = this.table.getFocus();
+    if (!cells.find((item) => item !== undefined) && typeof focus.position.row === 'undefined') {
```

Comment:
Sorry, I fail to see what `getAnswerGiven` in Crossword has to do with CoursePresentation. But yes, `getAnswerGiven` in CoursePresentation may not be what you're looking for there. And again: There's no publicly available documentation, and I don't know the hard rules that should determine whether that H5P.com restart button should show or not. That's why I asked about the focus, for instance. If the requirement is just some letter entered into the grid or text input fields, then that is what `getAnswerGiven` will tell you here.

I listed `cells.every` as a way to prevent double negation and to make the code clearer as mentioned, not to speed things up, and `cells.some` as an alternative to `cells.every` - also more clear than `find`, but with the same traversal logic that doesn't necessarily look at all the items.